client/app/dedicated/server/working-status/WORKS.html

```diff
@@ -0,0 +1,34 @@
+<div data-ng-controller="WorkingStatusCtrl as $ctrl">
+
+    <oui-back-button on-click="currentView.value = 'DASHBOARD'"
+                     data-title="{{:: tr('modal_working_status_title') }}">
+    </oui-back-button>
+
+    <div data-ng-repeat="work in $ctrl.worksDetails track by $index">
+        <h5>
+            <a target="_blank"
+               rel="noopener"
+               title="{{ work.title }} ({{ tr('core_new_window') }})"
```

Comment:
`tr` is not use anymore, you can use angular translate.

So, you can now use `translate-attr` :D